src/Symfony/Component/Validator/Mapping/Cache/DoctrineCache.php

```diff
@@ -30,6 +34,7 @@
      */
     public function __construct(Cache $cache)
     {
+        @trigger_error(sprintf('%s is deprecated since version 3.2 and will be removed in 4.0. Use %s with %s instead.', __CLASS__, Psr6Cache::class, DoctrineAdapter::class), E_USER_DEPRECATED);
```

Comment:
I agree but the deprecation is triggered by the ``AutowirePass`` as this class is defined as a service in the core.